src/main/java/com/eternalcode/parcellockers/ParcelLockersCommand.java

```diff
@@ -3,42 +3,39 @@
 import com.eternalcode.parcellockers.configuration.ConfigurationManager;
 import com.eternalcode.parcellockers.configuration.implementation.PluginConfiguration;
 import com.eternalcode.parcellockers.notification.NotificationAnnouncer;
-import dev.rollczi.litecommands.command.async.Async;
-import dev.rollczi.litecommands.command.execute.Execute;
-import dev.rollczi.litecommands.command.permission.Permission;
-import dev.rollczi.litecommands.command.route.Route;
-import net.kyori.adventure.text.minimessage.MiniMessage;
+import dev.rollczi.litecommands.annotations.async.Async;
+import dev.rollczi.litecommands.annotations.command.Command;
+import dev.rollczi.litecommands.annotations.context.Context;
+import dev.rollczi.litecommands.annotations.execute.Execute;
+import dev.rollczi.litecommands.annotations.permission.Permission;
 import org.bukkit.command.CommandSender;
 import org.bukkit.entity.Player;
 import org.bukkit.inventory.ItemStack;
 
-@Route(name = "parcellockers", aliases = {"parcellocker"})
+@Command(name = "parcellockers", aliases = { "parcellocker" })
```

Comment:
Why you change major version of lite commands in pull requests that mean "finish sending logic"